crates/ruma-identifiers/src/session_id.rs

```diff
@@ -0,0 +1,12 @@
+//! Matrix session ID.
+
+use ruma_identifiers_validation::session_id::validate;
+
+opaque_identifier_validated! {
+    /// A session ID.
+    ///
+    /// Session IDs in Matrix are opaque character sequences of `[0-9a-zA-Z.=_-]. Their length must
```

Comment:
Inline code block still unclosed.

```suggestion
    /// Session IDs in Matrix are opaque character sequences of `[0-9a-zA-Z.=_-]`. Their length must
```

Reply:
Sorry, I thought I had already fixed that one!